sematic/db/migrations/20221214142609_plugin_settings_file.py

```diff
@@ -0,0 +1,100 @@
+# Standard Library
+import os
+import pathlib
+from typing import Any, Dict
+
+# Third-party
+import yaml
+
+THIS_MIGRATION_SCHEMA_VERSION = 1
+
+
+def up():
+    config_dir_path = _get_config_dir()
+    user_settings_file_path = os.path.join(config_dir_path, "settings.yaml")
+    server_settings_file_path = os.path.join(config_dir_path, "server.yaml")
+
+    user_loaded_yaml = _load_settings_yaml("settings.yaml")
+    server_loaded_yaml = _load_settings_yaml("server.yaml")
+
+    schema_version = user_loaded_yaml.get("version", 0)
+
+    if schema_version != THIS_MIGRATION_SCHEMA_VERSION - 1:
+        raise RuntimeError("Exception ran in incorrect order")
+
+    new_settings = {
+        "version": THIS_MIGRATION_SCHEMA_VERSION,
+        "profiles": {
+            "default": {
+                "scopes": {},
+                "settings": {
+                    "sematic.config.user_settings.UserSettings": user_loaded_yaml.get(
+                        "default", {}
+                    ),
+                    "sematic.config.server_settings.ServerSettings": server_loaded_yaml.get(  # noqa: E501
+                        "default", {}
+                    ),
+                },
+            }
+        },
+    }
+
+    with open(user_settings_file_path, "w") as f:
+        f.write(yaml.dump(new_settings, Dumper=yaml.Dumper))
+
+    os.remove(server_settings_file_path)
+
+
+def down():
+    loaded_yaml = _load_settings_yaml("settings.yaml")
+
+    schema_version = loaded_yaml.get("version", 0)
+
+    if schema_version != THIS_MIGRATION_SCHEMA_VERSION:
+        raise RuntimeError("Exception ran in incorrect order")
```

Comment:
raise RuntimeError(f"Cannot downgrade settings file from version {schema_version} to version {THIS_MIGRATION_SCHEMA_VERSION}")

Reply:
Done.